.rubocop.yml

```diff
@@ -1,23 +1,18 @@
 inherit_from:
   - .rubocop_todo.yml
 
-require:
-  - rubocop-performance
-  - rubocop-rails
-  - rubocop-minitest
+inherit_gem:
+  theforeman-rubocop:
+    - strictest.yml
 
 AllCops:
-  TargetRubyVersion: 2.7
-  TargetRailsVersion: 5.2
   Exclude:
-    - 'extras/**/*'
-    - 'locale/**/*'
-    - 'node_modules/**/*'
```

Comment:
Similar to the other place: you need to keep these locations. It's causing the current build issue.